extra-i18n/ydcv-rs/autobuild/defines

```diff
@@ -1,7 +1,7 @@
 PKGNAME=ydcv-rs
 PKGDES="A Rust version of ydcv"
 PKGDEP="openssl dbus libxcb"
-BUILDDEP="git cargo python-3"
+BUILDDEP="git cargo python-3 llvm"
```

Comment:
Please switch `cargo` to `rustc`, as `cargo` is now a transitional package which will (eventually) be removed.

Reply:
Done